.github/workflows/black.yml

```diff
@@ -0,0 +1,20 @@
+name: Linting with black
+
+on:
+  push:
+  pull_request:
+      
+jobs:
+
+  black-linting:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v3
+
+      - name: Format code with black
+        run: |
+          pip install black
+          black . --exclude "main.py"
```

Comment:
May I know why we excluded the main.py file here >?

@metabiswadeep

Reply:
The main.py file has lines beginning with @app.get which the black linter cannot parse due to it not being Python code, and raises errors.